lib/engine/game/g_18_carolinas.rb

```diff
@@ -7,6 +7,57 @@ module Engine
   module Game
     class G18Carolinas < Base
       load_from_json(Config::Game::G18Carolinas::JSON)
+
+      def init_companies(players)
+        super.take(players.size).each do |company|
+        end
+      end
+
+      def setup
+        counter = 0
+        @companies.each do |company|
```

Comment:
@companies.each_with_index do |company, index| # instead of using counter

Reply:
fixed